src/setupTests.js

```diff
@@ -9,3 +9,28 @@ jest.mock('src/libs/ajax')
 jest.mock('src/libs/nav')
 
 window.Element.prototype['insertAdjacentElement'] = () => {} // for custom icons
+
+ReactWrapper.prototype.testId = function(id) {
```

Comment:
I don't like mutating this, but as we've discussed, there isn't really an alternative, unless @panentheos can think of something.